Qt/mainwindow.h

```diff
@@ -90,27 +90,37 @@ private slots:
 	void takeScreen() { g_TakeScreenshot = true; }
 	void consoleAct();
 
-	// Options
-	// Core
-	void vertexDynarecAct() { g_Config.bVertexDecoderJit = !g_Config.bVertexDecoderJit; }
-	void fastmemAct() { g_Config.bFastMemory = !g_Config.bFastMemory; }
-	void ignoreIllegalAct() { g_Config.bIgnoreBadMemAccess = !g_Config.bIgnoreBadMemAccess; }
-
-	// Video
-	void anisotropicGroup_triggered(QAction *action) { g_Config.iAnisotropyLevel = action->data().toInt(); }
+	// Game settings
+	void keepOnTopAct() { g_Config.bTopMost = !g_Config.bTopMost; }
+	void pauseWhenNotFocusedAct() { g_Config.bPauseOnLostFocus = !g_Config.bPauseOnLostFocus; }
```

Comment:
This one also needs hook-up with Qt.  On Windows, we listen for WM_ACTIVATE, and basically call `Core_EnableStepping(true/false);` to pause or resume emulation.

-[Unknown]